lib/zoneddatetime.ts

```diff
@@ -105,17 +106,17 @@ export class ZonedDateTime implements Temporal.ZonedDateTime {
   get epochSeconds(): Return['epochSeconds'] {
     if (!ES.IsTemporalZonedDateTime(this)) throw new TypeError('invalid receiver');
     const value = GetSlot(this, EPOCHNANOSECONDS);
-    return JSBI.toNumber(ES.BigIntFloorDiv(value, BILLION));
+    return JSBI.toNumber(BigIntFloorDiv(value, BILLION));
```

Comment:
This pattern is repeated many times. Should it be encapsulated into bigintmath.ts?

Reply:
:shrug: ... I think having BigIntFloorDiv in bigintmath.ts is DRY enough. We only use it a few times anyway, and only some of those want a Number result.